programs/marginfi/src/state/price.rs

```diff
@@ -79,7 +77,6 @@ pub trait PriceAdapter {
 #[enum_dispatch(PriceAdapter)]
 #[cfg_attr(feature = "client", derive(Clone))]
 pub enum OraclePriceFeedAdapter {
-    PythLegacy(PythLegacyPriceFeed),
     SwitchboardV2(SwitchboardV2PriceFeed),
```

Comment:
I think your approach removing the enum here is better, let's do the same for the switchboardv2 stub I left behind